src/background/responders.ts

```diff
@@ -67,6 +70,47 @@ export const responders: { [T in Action['type']]: Responder<T> } = {
       },
     }
   },
+  FETCH_HANDLE_SUCCESS(state, action): Partial<AppState> {
+    const { handle } = action.payload
+    const tab = activeTab(state)
+
+    // If the tab doesn't exist anymore, don't try to update it
+    if (tab === null) return {}
```

Comment:
Let's use the shorthand `if (!tab)`, which is easier to read and more resistant if the function returns `undefined | false`, neither of which we should act on